tools/base.py

```
@@ -1,6 +1,7 @@
 ## base.py
 from abc import ABCMeta, abstractmethod
 from dataclasses import dataclass, fields, replace
+from email import message
```

Ruff (0.8.2)

`email.message` imported but unused

Remove unused import: `email.message`

(F401)